mlflow/deployments/mlflow/__init__.py

```diff
@@ -7,7 +7,6 @@
 from mlflow.deployments.constants import (
     MLFLOW_DEPLOYMENT_CLIENT_REQUEST_RETRY_CODES,
 )
-from mlflow.deployments.server.config import Endpoint
```

Comment:
note: Ideally we should change the logic for how to capture mlflow[gateway] extra. Currently MLflow adds the extra whenever it sees importing "mlflow.gateway" module, which does not always mean the code starts the deployment server.